### DISCUSSION TITLE
Print customizing in the printpipline section ?

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#247] [Q&A] [answered]
would it not make more sence to move the printer enablement and the customizing from the hardwareinoutput to the mediaprocessing_pipeline_printing admin section ?

I think that is more intuitiv ?

**Comment by mgineer85** [comment, marked as answer]
There will be some refactoring in future to flexibilize the trigger -> action mapping. 

Not yet there :)

**Reply by Striker62** [reply]
Ok I'm courios how it will look like 👍
